aspects/models/problems/fact_learner_problem_summary.sql

```diff
@@ -0,0 +1,69 @@
+-- summary table for a learner's performance on and interactions with a
+-- particular problem
+with results_with_hints as (
+    select
+        org,
+        course_key,
+        problem_id,
+        actor_id,
+        success,
+        attempts,
+        0 as num_hints_displayed,
+        0 as num_answers_displayed
+    from {{ ref('int_problem_results') }}
+    union all
+    select
+        org,
+        course_key,
+        problem_id,
+        actor_id,
+        null as success,
+        null as attempts,
+        case help_type
+            when 'hint' then 1
+            else 0
+        end as num_hints_displayed,
+        case help_type
+            when 'answer' then 1
+            else 0
+        end as num_answers_displayed
+    from {{ ref('int_problem_hints') }}
+), summary as (
+    -- n.b.: there should only be one row per org, course, problem, and actor
+    -- in problem_results, so any(success) and any(attempts) should return the
+    -- values from that part of the union and not the null values used as
+    -- placeholders in the problem_hints part of the union
+    select
+        org,
+        course_key,
+        problem_id,
+        actor_id,
+        coalesce(any(success), false) as success,
+        coalesce(any(attempts), 0) as attempts,
+        sum(num_hints_displayed) as num_hints_displayed,
```

Comment:
I'm not 100% sure here, but I think the answers and hints are cumulative across events, so summing them here would effectively double them. Can you double check?

Reply:
Do you mean that there's a field that tracks number of hints shown, similar to how `attempts` will keep a running count of number of attempts? If so, I don't think we'd be double counting, since we're counting the number of hint/answer shown events, and not summing up a particular field.

Reply:
Oh I see, sorry I missed that 👍